inst/rmarkdown/templates/isba/skeleton/fig001.eps

```diff
@@ -0,0 +1,524 @@
+%!PS-Adobe-2.0 EPSF-2.0
```

Comment:
I don't think we need this file. It is not use in the Rmd template. It is used in `badoc.tex` and we don't need that file I think. This should be remove IMO.

Same for the associated pdf file.